ocs_sample_library_preview/Asset/Status/StatusEnum.py

```diff
@@ -0,0 +1,11 @@
+from enum import Enum
+
+
+class StatusEnum(Enum):
+    """
+    enum 0-3  fully inclusive
```

Comment:
There are two spaces between "3" and "fully". This is nitpicky and maybe this was intentional, but I figured I should bring it up just in case.

Reply:
Made this consistent (just one space)